### PR TITLE
Dynamic path fix, slackActions, starter app

### PR DESCRIPTION
**Pull request by akhil-raj-sfdc**
Edited package.json and tsconfig to fix existing path errors.

Updated certain commonjs imports to ES6+

Created a starter app

To-Do: 
- Add starter app manifest to repo
- Update README with instructions
- Set repo as a template

### CODE REVIEW

File: src/actions/click-me-modal-open.ts [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  constructor(app: App) {
+    this.app = app;
+
+    this.listen();
+  }
+
+  listen() {
+    this.app.action(
+      {
+        type: Constants.BLOCK_ACTIONS,
```
Comment (dbylerSFDC) [inline src/actions/click-me-modal-open.ts:20]: Do you mind sharing your thought process about abstracting the type to a constant?

File: tsconfig.json [hunk 2]
```diff
@@ -12,14 +12,26 @@
...
+    // Customize or add to these paths as needed
     "baseUrl": "src",
     "paths": {
-      "$components/*": ["views/components/components.ts"],
-      "$modals/*": ["views/modals"],
-      "$messages/*": ["views/messages"],
-      "$home-tab/*": ["views/home-tab"],
-      "$emojis/*": ["utilities/emojis.ts"],
-      "$constants/*": ["utilities/constants.ts"]
+      "$actions": ["actions"],
```
Comment (dbylerSFDC) [inline tsconfig.json:18]: I'm curious how this works. The first line is access to the Actions index file and the second is if you need access to only a specific action file?

**Review by dbylerSFDC — APPROVED** [review]
Love it! This is great :D

File: src/actions/click-me-modal-open.ts (hunk 1 above)
Comment (akhil-raj-sfdc) [reply src/actions/click-me-modal-open.ts:20]: No big reason to do so. Often find myself using the 'block_actions' type. To reduce spelling error, I figured it's easier to call BLOCK_ACTIONS from the Constants class.

File: tsconfig.json (hunk 2 above)
Comment (akhil-raj-sfdc) [reply tsconfig.json:18]: Yup that's correct.

File: src/actions/click-me-modal-open.ts (hunk 1 above)
Comment (dbylerSFDC) [reply src/actions/click-me-modal-open.ts:20]: Ok, cool! Yeah, that makes sense, then it's autofilled

**Pull request merged**